sdks/nodejs/package.json

```diff
@@ -26,5 +26,5 @@
     "publishConfig": {
         "access": "public"
     },
-    "version": "1.14.0"
+    "version": "1.15.0-dev"
```

Comment:
If editing manually we should run `npm install` after changing this so that `package-lock.json` is also updated

There is a useful command we can run instead:
```
npm version preminor --preid=dev
```
This would bump the version to `1.15.0-dev.0` - close enough?